src/testlib/include/testlib/clocks.hpp

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019-present MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef HEADER_1112A012_F4B7_4C67_9141_7B1179B788CC_INCLUDED
+#define HEADER_1112A012_F4B7_4C67_9141_7B1179B788CC_INCLUDED
+
+#include <chrono>
+
+#include <metrics/Period.hpp>
+
+namespace genny::testing {
+using namespace std::literals::chrono_literals;
+
+// The template is to allow different DummyClocks classes to be created so calling static methods
+// won't conflict.
+template <typename DummyT>
+struct DummyClock {
```

Comment:
The changes to this file is unrelated to the feature but I think it's worth putting clocks together before we write another clock for testing sleep-related functionality